.github/workflows/tests.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - master
   pull_request:
+    paths-ignore:
+    - README.md
```

Comment:
Actually, no, this will fail because the CI job will still be marked as `required` but just won't be run 😢